directus-cms/extensions/directus-extension-programmierbar-bundle/src/create-profile/index.ts

```diff
@@ -0,0 +1,62 @@
+import { defineHook } from '@directus/extensions-sdk'
+import { createHookErrorConstructor } from '../shared/errors.ts'
+
+const HOOK_NAME = 'create-profile'
+
+export default defineHook(({ filter }, hookContext) => {
+    const logger = hookContext.logger
+    const ItemsService = hookContext.services.ItemsService
+
+    filter('users.create', (payload, metadata, context) => handleFilter( { payload, metadata, context }))
```

Comment:
Ich frage mich hier gerade, ob wir das in keinem Hook typisiert haben? Irgendwie fällt es mir schwer da durchzublicken, was alles in den Payloads drin ist 😅  

Aber scheint nicht so zu sein und du bist ja positiv, dass das alles läuft, von daher wird das vermutlich auch korrekt sein, was hier alles gemacht wird :D

Reply:
Adressiert! Danke!